content/project/index.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: simplepage
+title: "Project structure and governance"
+section: project
+---
+
+The Jenkins project as a community revolves around Jenkins as a piece of software.
+We are a group of open-source developers and users who develop, use, promote Jenkins, software around Jenkins, and other related activities for our mutual benefit.
+
+This page provides some links to pages describing the project structure and its governance.
+
+### Project governance
+
+* link:./governance[Jenkins Governance Document] - overview of the project's governance model, philosophy, project roles, decision making process, etc.
+* link:./conduct[Code of Conduct]
+* link:./board-election-process[Jenkins Board election process]
+
+### Project structure
+
+* link:./board[Governance Board members and current officers]
+* link:./team-leads[Team leaders]
+* Teams:
+** link:/security/#team[Security Team]
```

Comment:
repeating team in the name sounds unnecessary? it's not in the other one and it's the name of the list
```suggestion
** link:/security/#team[Security]
```